src/Records/src/Student.java

```diff
@@ -0,0 +1,15 @@
+ public record Student(String name, int Class, int ClassNum) {
```

Comment:
```suggestion
 public record Student(String name, int class, int classNum) {
```
variable names begin in lowercase letters